src/chords_generation.py

```diff
@@ -20,14 +24,31 @@ def get_tonation_chord(tonation: music.Tonation, sound: music.Sound):
     return tonation.tonic
 
 
-def get_chords(sounds: List[music.Sound], tonation: music.Tonation, metrum: int, start: int):
-    sounds_at_metrum = get_sounds_at_metrum(sounds, metrum, start)
+def get_chords(sounds: List[music.Sound],
+               tonation: music.Tonation,
+               meter: Tuple[int, int]):
+    meter = (
+        meter[0],
+        utils.constants.RHYTHMIC_VALUES[str(meter[1])]
+    )
+    if meter[0] == 2:
+        # make 2 times more notes with 2 times less time duration each
+        meter = (
+            4,
+            meter[1]/2
+        )
+    if meter[0] != 4:
+        raise Exception("Not supported meter in chord duration")
+    
+    sounds_at_metrum = get_sounds_at_metrum(sounds, meter)
     chords = [get_tonation_chord(tonation, sound)
               for sound in sounds_at_metrum]
-    
+
     c1 = []
+    # chord_duration = half of meter * how many eigth fits in one meter note
+    chord_duration = (meter[0]/2) * (meter[1]/0.5)
```

Comment:
shouldn't it be `(meter[1]/2) * (meter[0]/0.5)` ?
I mean the result is the same, but the comment makes more sense

Reply:
meter 4/8 means there are 4 eigths in one bar. From that chord duration is 2 (half of meter) *  1 (number of eigths in eigth)

Reply:
Ok, you're right